bin/annotate_test_failures

```diff
@@ -100,6 +115,76 @@ def update_annotation(title, list, style, state)
   end
 end
 
+# Given a list of failures, parse list and send a slack notification with the test names in the payload
+#
+def send_slack_notification(slack_webhook, slack_channel, title, list)
+  tests = list.map do |item|
+    {
+      'name' => item.name,
+      'classname' => item.classname,
+      'message' => item.message,
+      'details' => item.details
+    }
+  end
```

Comment:
I'm not sure I'm seeing the point of converting the array of `TestFailure` objects into an array of Hashes/Dictionaries like that, only to have to use `test['name']` and the likes to access each Hash's key in the code below… instead of directly looping on `list` on the code below then using `test.name` to access each TestFailure's property directly?

I'm guessing this idea is more of a leftover from the fact that your previous implementation was using an intermediate json? But now that it's not and directly receives the array of `TestFailure` objects I think you can get rid of those lines, and have the rest of the code directly use the `list` variable instead of `tests`